gossip/src/duplicate_shred_handler.rs

```diff
@@ -1,95 +1,59 @@
 use {
     crate::{
-        duplicate_shred::{into_shreds, DuplicateShred, Error},
+        duplicate_shred::{self, DuplicateShred, Error},
```

Comment:
confused, why does it need more than into_shreds now?

Reply:
By convention in general it is better to `use` module name instead and use the qualified name for functions for more clarity.